delphi/translators/for2py/f2grfn.py

```diff
@@ -473,7 +473,7 @@ def fortran_to_grfn(
         with open(original_fortran_file_path, "r") as f:
             inputLines = f.readlines()
     except IOError:
-        assert False, "Fortran file: {original_fortran_file_path} Not Found"
+        assert False, f"Fortran file: {original_fortran_file_path} Not Found"
```

Comment:
Since execution only gets to this point because an exception has been raised --- i.e., we know that things have gone wrong --- an `assert` seems not quite the right construct here.  The following seems conceptually more straightforward to me:
```
sys.stderr.write( ... )
sys.exit(1)
```